### PR TITLE
Fix findbugs errors

### PR DESCRIPTION
**Pull request by robinwithes**
Used https://plugins.jetbrains.com/plugin/3847-findbugs-idea to look for potentially problematic code and tried to amend it. Also did some refactoring mainly in SimplePreferences to make it look more structured

### CODE REVIEW

File: src/main/java/com/stirante/RuneChanger/client/ChampionSelection.java [hunk 1]
```diff
@@ -116,7 +116,7 @@ private void findSelectedChampion(LolChampSelectChampSelectSession session) {
                             //noinspection unchecked
                             Map<String, Object> selectAction = (Map<String, Object>) obj;
                             if (selectAction.get("type").equals("pick") &&
-                                    selectAction.get("actorCellId") == selection.cellId) {
+                                    selectAction.get("actorCellId").equals(selection.cellId)) {
```
Comment (stirante) [inline src/main/java/com/stirante/RuneChanger/client/ChampionSelection.java]: Have you checked if this works?

File: src/main/java/com/stirante/RuneChanger/RuneChanger.java [hunk 2]
```diff
@@ -287,7 +287,7 @@ public void onEvent(ClientWebSocket.Event event) {
                             } catch (IOException e) {
                                 e.printStackTrace();
                             }
-                        }).run();
+                        }).start();
```
Comment (stirante) [inline src/main/java/com/stirante/RuneChanger/RuneChanger.java:290]: Good catch

File: src/main/java/com/stirante/RuneChanger/RuneChanger.java [hunk 3]
```diff
@@ -211,7 +211,7 @@ public void onClientConnected() {
                         }
                         return;
                     }
-                }).run();
+                }).start();
```
Comment (stirante) [inline src/main/java/com/stirante/RuneChanger/RuneChanger.java:214]: Good catch

**Review by stirante — CHANGES_REQUESTED** [review]

File: src/main/java/com/stirante/RuneChanger/client/ChampionSelection.java (hunk 1 above)
Comment (robinwithes) [reply src/main/java/com/stirante/RuneChanger/client/ChampionSelection.java]: Not sure if it would but since they are both longs i reverted this change.

**Review by stirante — APPROVED** [review]

**Pull request merged**